content/applications/websites/ecommerce/getting_started.rst

```diff
@@ -1,11 +0,0 @@
-:nosearch:
-
-=================
-Get started
-=================
-
-.. toctree::
-   :titlesonly:
-
-   getting_started/product_page
-   getting_started/catalog
```

Comment:
Shouldn't there be a redirect rule to `ecommerce/managing_products.rst` for these two documents?

Reply:
right. I forgot :/ my bad

Reply:
Beware that runbot will be less polite when we delegate this check to him. :stuck_out_tongue:

Reply:
Yes, but I won't lose runbot's respect...

Reply:
![image](https://user-images.githubusercontent.com/15896813/216110369-db3841da-6a77-4a99-9064-517efc8c6cc8.png)